internal/cloudfoundry/setup.go

```diff
@@ -322,6 +329,10 @@ func (s Setup) Run(log io.Writer, home, name, source string) (string, error) {
 		args = append(args, "-b", buildpack)
 	}
 
+	if s.startCommand != "" {
+		args = append(args, "-c", fmt.Sprintf(`"%s"`, s.startCommand))
```

Comment:
This argument shouldn't need to be quoted. When `os/exec` handles the argument it will make sure that argument is quoted correctly. You can see the difference in behavior here: https://go.dev/play/p/rBvJTeTxKoE

Reply:
Resolved, thanks!